packages/security/.gitignore

```diff
@@ -0,0 +1,23 @@
+.DS_Store
```

Comment:
Hier kan er wel 1 van overblijven in de root?

Reply:
ja kan wellicht , maar laten we dat na de bump naar 12 doen, is niet afhankelijk van de node versie